release.yml

```diff
@@ -0,0 +1,123 @@
+# JIRA Project code where the release issue should be created
+jiraProject:     SB
+
+# If already created, provide the issue number of the Release, otherwise keep it empty.
+# It will be used by the command create-component to link the stakeholder issues
+jiraKey:         # SB-1484
+
+# Version to be released. This version will be used to populate the description to be created for each Jira stakeholder: component or starter owner
+version:         2.3.0.RELEASE
+
+# Long version name used to create the title of the release issue
+longVersionName: "[Spring Boot 2.3.0] Release steps CR [Q3-2020]"
```

Comment:
Do we really need this? Shouldn't that be inferred from `version` and `releaseDate`?

Reply:
Yes as this is the text that we will add to the title of the jira cloned release issue. PR is welcome to improve how we could populate it using a text template.